### PR TITLE
QC fixes 

### PR DESCRIPTION
**Pull request by tannermpeterson**

### CODE REVIEW

File: frontend/pulse3d/src/components/interactiveAnalysis/BasicWaveformGraph.js [hunk 1]
```diff
@@ -100,40 +128,45 @@ export default function BasicWaveformGraph({ well, featureIndices, waveformData,
...
-      .data(peaks)
+      .data(peaksWithinWindow)
       .enter()
       .append("path")
       .attr("id", "peak")
-      .attr("indexToReplace", (d) => peaks.indexOf(d)) // keep track of index in peaks array to splice later
       .attr("d", d3.symbol().type(d3.symbolTriangle).size(50))
       .attr("transform", (d) => {
-        return "translate(" + x(waveformData[d][0]) + "," + (y(waveformData[d][1]) - 7) + ") rotate(180)";
+        console.log("idx", d);
```
Comment (luciipak) [inline frontend/pulse3d/src/components/interactiveAnalysis/BasicWaveformGraph.js:137]: console.log

File: frontend/pulse3d/src/components/interactiveAnalysis/InteractiveWaveformGraph.js [hunk 2]
```diff
@@ -186,13 +187,23 @@ const contextMenuItems = {
 const NoFeaturesAlert = styled.div`
   color: red;
 `;
+
+const applyWindow = (data, xMin, xMax) => {
```
Comment (luciipak) [inline frontend/pulse3d/src/components/interactiveAnalysis/InteractiveWaveformGraph.js:191]: Might be worth later adding to utils

**Comment by luciipak** [comment]
Cna we bump the FE version? I want to see why it's erroring during the upload

File: frontend/pulse3d/src/components/interactiveAnalysis/InteractiveWaveformGraph.js (hunk 2 above)
Comment (tannermpeterson) [reply frontend/pulse3d/src/components/interactiveAnalysis/InteractiveWaveformGraph.js:191]: Yeah, this can definitely be cleaned up

**Comment by tannermpeterson** [comment]
Yeah I'll make a PR for all of this

**Review by luciipak — APPROVED** [review]

**Pull request merged**